.github/workflows/lint.yml

```diff
@@ -9,4 +9,5 @@ jobs:
       with:
         node-version: 12.x
     - run: npm ci
+    - run: npm run test
```

Comment:
I'll send a PR to rename this workflow to reflect that it now does more than linting.

Reply:
Actually, it still only concerns linting, I'll leave it for now.